core/src/main/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisher.java

```diff
@@ -160,6 +160,25 @@ public void setAdditionalExceptionMappings(Properties additionalExceptionMapping
 		}
 	}
 
+	/**
+	 * Overload of {@link #setAdditionalExceptionMappings(Properties)}}, providing the ability
+	 * to map authentication exceptions and failure authentication events using a {@link Map}
+	 *
+	 * @param mappings where keys are exception classes and values are event classes.
+	 */
+	public void setAdditionalExceptionMappings(Map<Class<? extends AuthenticationException>,
+			Class<? extends AbstractAuthenticationFailureEvent>> mappings){
+		Assert.notEmpty(mappings, "The mappings Map must not be empty nor null");
+		for (Map.Entry<Class<? extends AuthenticationException>,Class<? extends AbstractAuthenticationFailureEvent>> entry
```

Comment:
The build should have alerted you to a formatting change that it wants you to do on this line. Does `./gradlew clean check` work for you to pick up these requirements?